graylog2-server/src/test/java/org/graylog2/indexer/cluster/ClusterIT.java

```diff
@@ -226,4 +247,19 @@ public void getDefaultClusterAllocationDiskSettings() {
         assertThat(clusterAllocationDiskSettings.watermarkSettings().high()).isEqualTo(90D);
         assertThat(clusterAllocationDiskSettings.watermarkSettings().floodStage()).isEqualTo(95D);
     }
+
+    @Test
+    public void getIndexShardsInfo() {
```

Comment:
@mpfz0r  or should I just straight away start an IndicesIT so that we have a dedicated place in the future?

Reply:
@kodjo-anipah there already is an IndicesIT :-D 
feel free to move it there if you think that's a better fit